starter_code/index.js

```diff
@@ -1 +1,4 @@
 const Elevator = require('./elevator.js');
+const Person = require('./person.js');
+
+let requester = new Person('Juan', 0, 3);
```

Comment:
this let should be a const. Try to start using const everywhere untill it complains